etc/kayobe/environments/ci-multinode/kolla/inventory/group_vars/all

```diff
@@ -0,0 +1,6 @@
+---
+
+# Just adding for testing
+enable_firewalld: true
+firewalld_zones: "{{ stackhpc_firewalld_zones }}"
+firewalld_rules: "{{ stackhpc_firewalld_rules }}"
```

Comment:
Will this be removed before merging. Or at least the comment reworded.

Reply:
For sure. I only added it cause I was hoping that it would apply the rules everywhere automatically when I deployed a multinode from this branch. Sadly it didn't turn out like that anyway.